.changeset/selfish-steaks-behave.md

```diff
@@ -0,0 +1,20 @@
+---
+"@hashicorp/design-system-components": major
+"@hashicorp/design-system-tokens": major
+---
+
+IMPORTANT: BEFORE RELEASING DOUBLE CHECK AND REVIEW WHAT'S WRITTEN BELOW
+
+# Form controls components
+
+- Added the form "text input" controls: `<Hds::Form::TextInput::Base>`, `<Hds::Form::TextInput::Field>`
+- Added the form "textarea" controls: `<Hds::Form::Textarea::Base>`, `<Hds::Form::Textarea::Field>`
+- Added the form "select" controls: `<Hds::Form::Select::Base>`, `<Hds::Form::Select::Field>`
+- Added the form "checkbox" controls: `<Hds::Form::Checkbox::Base>`, `<Hds::Form::Checkbox::Field>`, `<Hds::Form::Checkbox::Group>`
+- Added the form "radio" controls: `<Hds::Form::Radio::Base>`, `<Hds::Form::Radio::Field>`, `<Hds::Form::Radio::Group>`
+- Added the form "toggle" controls: `<Hds::Form::Radio::Base>`, `<Hds::Form::Radio::Field>`, `<Hds::Form::Radio::Group>`
+- Added the form "base" low-level elements: `<Hds::Form::Label>`, `<Hds::Form::HelperText>`, `<Hds::Form::Error>`, `<Hds::Form::Legend>`, `<Hds::Form::Field>`, `<Hds::Form::Fieldset>`
+
+# Design tokens
+
+- TODO add design tokens for components, if we end upp adding them for the form controls
```

Comment:
@didoo did we end up adding these?

Reply:
yes. good point, updating the changeset now.